### ISSUE TITLE
Duplicate entry: META-INF/services/com.smaato.sdk.core.framework.ModuleInterface

### ISSUE THREAD

**Issue by PromanSEW** [appodeal/appodeal-android-demo#41]
I updated Appodeal to 2.5.10, Android Studio 3.5.2, and get following error when building APK:
```
> Task :app:kaptGenerateStubsReleaseKotlin FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:kaptGenerateStubsReleaseKotlin'.
> Could not resolve all artifacts for configuration ':app:releaseCompileClasspath'.
   > Failed to transform artifact 'nodex.aar (com.appodeal.ads:nodex:2.5.10)' to match attributes {artifactType=android-classes, org.gradle.usage=java-api}.
      > Execution failed for AarToClassTransform: C:\Users\PromanSEW\.gradle\caches\transforms-2\files-2.1\54e2e78555bbdeb57dc4e5a99f7fe627\jetified-nodex-2.5.10.aar.
         > duplicate entry: META-INF/MANIFEST.MF
   > Failed to transform artifact 'smaato.aar (com.appodeal.networks:smaato:21.1.3)' to match attributes {artifactType=android-classes, org.gradle.usage=java-api}.
      > Execution failed for AarToClassTransform: C:\Users\PromanSEW\.gradle\caches\transforms-2\files-2.1\bfef99520ea144a625cfed9121940813\jetified-smaato-21.1.3.aar.
         > duplicate entry: META-INF/services/com.smaato.sdk.core.framework.ModuleInterface
```

My dependencies (worked fine for 2.5.9):
```gradle
implementation("com.appodeal.ads:nodex:2.5.10") {
   exclude module: "gson"
   exclude module: "okhttp"
   exclude module: "logging-interceptor"
   exclude module: "okio"
   exclude module: "retrofit"
   exclude module: "converter-gson"
}
implementation "com.google.android.exoplayer:exoplayer-core:2.8.4"
implementation "com.google.android.exoplayer:exoplayer-hls:2.8.4"
implementation "com.google.code.gson:gson:2.8.6"
implementation "com.squareup.retrofit2:retrofit:$retrofitVersion"
implementation "com.squareup.retrofit2:converter-gson:$retrofitVersion"
implementation "com.squareup.okhttp3:okhttp:$okhttpVersion"
implementation "com.squareup.okhttp3:logging-interceptor:$okhttpVersion"
implementation "com.squareup.okio:okio:$okioVersion"
```
Invalidate caches and restart, and clean project are not fixing this error

UPD: Android Studio 3.5.2 broke the build, not Appodeal update

**Comment by PromanSEW** [comment]
Reverted to 2.5.9 - error is still happens :(
Android Studio 3.5.2

**Comment by da2gl** [comment]
@PromanSEW Hello. What version of gradle plugin are you using?

**Comment by PromanSEW** [comment]
@da2gl Gradle 5.6.1, AS 3.5.2

**Comment by NikitaCherep** [comment]
@PromanSEW 
Try to use: classpath 'com.android.tools.build:gradle:3.5.1', 
instead classpath 'com.android.tools.build:gradle:3.5.2'

**Comment by PromanSEW** [comment]
@NikitaCherep Thanks, project build works fine now. Well, have to be at 3.5.1 before 3.6.0 release...

**Comment by da2gl** [comment]
@PromanSEW pay attention to this topic - https://issuetracker.google.com/issues/142675547.
Also, **we strongly recommend that you don't exclude libraries from our SDK and don't use other versions of these libraries.**

**Comment by PromanSEW** [comment]
Fixed in Android Studio 3.5.3
